src/Messenger/IndexationRequestHandler.php

```diff
@@ -54,6 +55,9 @@ public function __construct(Client $client, MessageBusInterface $bus, DocumentEx
         $this->client->setLogger(new NullLogger());
     }
 
+    /**
+     * @throws ExceptionInterface
+     */
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏We only call `schedule()` here, that method already have a `@throw`, are you sure we need to put this annotation on the whole chain of calls!?